02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+        "all_retrieved_cards": {
+            "Can children under 18 years of age ever be certified as a separate household from other adults living in the home?": {
+                "score_sum": 0.7314764598621692,
+                "derived_questions": [
+                    "Does the grandchild live with you?",
+                    "Do you provide regular support for the grandchild?",
+                    "Is the grandchild financially dependent on you?"
+                ],
+                "entire_card": "Can children under 18 years of age ever be certified as a separate household from other adults living in the home?\nAnswer:\nYes, if the adults with whom they are living do not exercise parental control or offer financial support. Clients who are officially emancipated usually fall under this category. If the client not entirely certain or it's close but messy we can error on the side of applying and inform the client the county will help them figure out the household if need be.\nExample: Ms. G is 17 years old. She lives with her aunt. Ms. G works and purchases and prepares her own meals. When questioned about household composition she stated that her aunt does not exercise parental control over her. The case manager certifies Ms. G as a one-person household.\nFood stamps only",
+                "summary": "Children under 18 years of age can be certified as a separate household from other adults living in the home if the adults do not exercise parental control or offer financial support, and if the child is officially emancipated."
+            },
+            "Who are mandatory HH members for food stamps?": {
+                "score_sum": 0.6699078276008735,
+                "derived_questions": [
+                    "Does the grandchild live with you?",
+                    "Do you provide regular support for the grandchild?",
+                    "Is the grandchild financially dependent on you?"
+                ],
+                "entire_card": "Who are mandatory HH members for food stamps?\nAnswer:\nThe following household members are ALWAYS included, even if they report that they do not share/prepare meals together. (see text only below, see flow chart further down)\nSpouses:\nIf living in the same residence\nChildren:\nIf age 17 or younger, the child is mandatory with legal guardian\nIf age 18-21, the child is mandatory if natural, adopted, or step child*\nA mandatory child's own child or spouse would need to be included on the application as well\n*Ex. If a 20 year old child resides with their parent, they would be mandatory.\n*Ex. If a 20 year old child resides with their grandparent, they would NOT be mandatory. They would likely be sharing/preparing meals together, but are not required to be listed on the same SNAP application.\n\ufeff\n\ufeff",
+                "summary": "The grandchild living with the client should be included as a mandatory household member for food stamps, regardless of whether they share food or not."
```

Comment:
Incorrect summary given the card's text: `*Ex. If a 20 year old child resides with their grandparent, they would NOT be mandatory. They would likely be sharing/preparing meals together, but are not required to be listed on the same SNAP application.` I presume this is an 'example' as opposed to an 'exception'. But it's the only sentence that mentions a grandparent/grandchild relationship. The card could be better written to be treated as a rule rather than an example.

Reply:
Relevance to the question: 🟢 Directly relevant
Correctness: 🔴 100% incorrect
Helpfulness: 🔴 Potentially harmful (leads the user down the wrong path)